src/utils/invoice.ts

```diff
@@ -23,6 +23,12 @@ export const liquidTestnetPrefix = "liquidtestnet:";
 
 export const maxExpiryHours = 24;
 
+const bolt11Prefixes = {
+    mainnet: "lnbc",
+    testnet: "lntb",
+    regtest: "lnbcrt",
+};
```

Comment:
Why configure those here?

Reply:
Keeps the config smaller

Reply:
Oh, come on. We can serve those couple characters

Reply:
Not about the characters, but it doesnt make sense to have it as a configurable value since it depends entirely on the network.

Reply:
Not about the characters, but it doesnt make sense to have it as a configurable value since it depends entirely on the network.